blocks/advantages/__picture/advantages__picture.css

```diff
@@ -0,0 +1,12 @@
+.advantages__picture{
+  width: 100%;
+  height: 100%;
+  object-fit: cover;
+  object-position: top center;
+}
+
+@media screen and (max-width: 1199px) {
+    .advantages__pictures :not(:last-of-type){
```

Comment:
Hey @NazarPauk,

Could we apply `:not(:last-of-type) { display: none; }` to the element with the class name `advantages__picture-wrapper` instead? Thanks! 🙏

Reply:
Done✅